script/mockbigscreenplayer.js

```diff
@@ -264,6 +267,9 @@ define('bigscreenplayer/mockbigscreenplayer',
       getPlayerElement: function () {
         return;
       },
+      getFrameworkVersion: function () {
+        return version;
```

Comment:
You can probably not bother mocking this out and introduce Version as a dependency for direct usage like in bigscreenplayer.js

Reply:
Thank you. I changed it.